.github/workflows/ci-actions.yml

```diff
@@ -80,7 +80,7 @@ jobs:
           restore-only: ${{ github.event_name == 'pull_request' }}
       - name: Build
         run: |
-          mvn -e -B  -DskipTests=true -Dno-format -DskipDocs clean install
+          mvn -e -B  -DskipTests=true -Dformat.skip -DskipDocs clean install
```

Comment:
I don't think we want to switch that particular one because I suppose `format.skip` disables the formatter entirely?

We need the first build to test the format and report the errors.